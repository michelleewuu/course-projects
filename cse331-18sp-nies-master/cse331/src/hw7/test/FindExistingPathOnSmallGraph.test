# Tests find path on smallGraph.

# Load smallGraph.tsv
LoadGraph smallGraph smallGraph.tsv

# Find minimum-cost path from NodeA to NodeB and from NodeB to NodeA
FindPath smallGraph NodeA NodeB
FindPath smallGraph NodeB NodeA