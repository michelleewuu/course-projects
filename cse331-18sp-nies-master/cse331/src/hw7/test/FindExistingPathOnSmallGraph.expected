# Tests find path on smallGraph.

# Load smallGraph.tsv
loaded graph smallGraph

# Find minimum-cost path from NodeA to NodeB and from NodeB to NodeA
path from NodeA to NodeB:
NodeA to NodeB with weight 1.000
total cost: 1.000
path from NodeB to NodeA:
NodeB to NodeA with weight 1.000
total cost: 1.000